h/static/styles/partials-v2/_pager.scss

```diff
@@ -0,0 +1,36 @@
+.pager {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin-top: 45px;
+  font-weight: bold;
+  justify-content: center;
+}
+
+.pager__item {
+  border-radius: 2px;
+  color: $grey-6;
+  padding: 7px 10px 7px 10px;
+}
+
+.item-highlight {
```

Comment:
The `item-*` classes should be renamed to fit out conventions:

`component__element` for elements
`is-$state` for state modifiers, in which case they should always be written as a combination selector along with the element that they style, eg `.pager__item.is-highlighted { ... }` rather than `is-highlighted { ... }`. This avoids the risk that they apply to the wrong element.